src/rdysetup.c

```diff
@@ -474,13 +473,13 @@ PetscErrorCode InitBoundaries(RDy rdy) {
         RDySurfaceCompositionSpec surface_comp = rdy->config.surface_composition[isurf_comp];          \
         if (!strcmp(surface_comp.region, region.name)) {                                               \
           if (mat_prop_vec) {                                                                          \
-            PetscScalar *x_ptr;                                                                        \
-            PetscCall(VecGetArray(mat_prop_vec, &x_ptr));                                              \
+            PetscScalar *u_ptr;                                                                        \
+            PetscCall(VecGetArray(mat_prop_vec, &u_ptr));                                              \
```

Comment:
Since `mat_prop_vec` is a 1 DOF vector, how about renaming `u_ptr` to `u1d_ptr` (or something else) that makes it a bit explicit that this is a pointer to 1 DOF vector.

Reply:
How about `prop_ptr`?

Reply:
That sounds good.